news/2020-11-03-new-featured-artist-se-u-ra.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "New Featured Artist: Se-U-Ra"
+date: 2020-11-03 09:00:00 +0000
+---
+
+We're excited to welcome **Se-U-Ra** aboard as our latest addition to the Featured Artist line-up!
+
+![](https://assets.ppy.sh/artists/112/header.jpg)
+
+No stranger to the rhythm gaming scene, **Se-U-Ra** is a Japanese artist and producer who has appeared in games like *Dynamix* and *K-Shoot Mania* and is a regular participant in the *BMS* community.
+
+With a whole new spin on the sound of a scene we all know and love, we're sure you'll find something to pique your interest in this latest selection of **38** new tracks, plus a showcase map from the masterminds of the Mappers' Guild.
+
+Check it all out:
+
+### Qlifact -another-
+
+Check out [this showcase set](https://osu.ppy.sh/beatmapsets/1224699) hosted by [Nao Tomori](https://osu.ppy.sh/users/5364763) of the Mappers' Guild, including [Icekalt](https://osu.ppy.sh/users/5410645)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/112/previews/2940.mp3" type="audio/mpeg">
+</audio>
+
```

Comment:
Check out [this osu!mania showcase set](https://osu.ppy.sh/beatmapsets/1226466) hosted by [FAMoss](https://osu.ppy.sh/users/7707789) of the Mappers' Guild!